FFmpeg/FFmpegFile.h

```diff
@@ -99,6 +99,50 @@ namespace OFX {
 class ImageEffect;
 }
 
+class MyAVPacket
+{
+public:
+    MyAVPacket(AVPacket *avpkt = nullptr)
+    {
+        InitPacket(avpkt);
+    }
+    ~MyAVPacket()
+    {
+        FreePacket();
+    }
+
+public:
+    void InitPacket(AVPacket *avpkt)
```

Comment:
these two methods should not be public. move the content of InitPacket to the constructor, and the content of FreePacket to the destructor.